conf/solr/crawldb/conf/managed-schema

```diff
@@ -116,20 +116,48 @@
 
+   <!-- Deprecated Fields -->
+   <!-- <field name="jobId" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> -->
+   <!-- <field name="lastFetchedAt" type="date" indexed="true" stored="true" multiValued="false" docValues="true" /> -->
+   <!-- <field name="numTries" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="numFetches" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="status" type="status" indexed="true" stored="true" default="NEW" multiValued="false" required="true" docValues="true" /> -->
+   <!-- <field name="depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="plainText" type="text_general" indexed="true" stored="true" multiValued="false" /> -->
+
+   <!-- Proposed Sparkler Schema -->
+   <field name="fetch_timestamp" type="date" indexed="true" stored="true" multiValued="false" docValues="true" /> 
+   <field name="content_type" type="string" indexed="true" stored="true" multiValued="false" docValues="true" /> 
+   <field name="crawler" type="string" indexed="true" stored="true" multiValued="false" default="sparkler"/> 
+   <field name="crawl_id" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> 
+   <field name="raw_content" type="text_general" indexed="true" stored="true" multiValued="false" />
+   <field name="status" type="status" indexed="true" stored="true" default="UNFETCHED" multiValued="false" required="true" docValues="true" />
+   <field name="extracted_text" type="text_general" indexed="true" stored="true" multiValued="false" />
+   <field name="fetch_status_code" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
+   <field name="hostname" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
+   <field name="signature" type="string" indexed="true" stored="true" multiValued="false" docValues="true" />
+   <field name="version" type="float" indexed="true" stored="true" required="true" default="1.0" multiValued="false" docValues="true" />
+   <field name="outlinks" type="string" indexed="true" stored="true" multiValued="true" />
+   <field name="crawler_discover_depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
+   <field name="crawler_fetch_depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
+   <field name="relative_path" type="string" indexed="true" stored="true" multiValued="false" docValues="true" />
+   <field name="parent" type="string" indexed="true" stored="true" multiValued="false" docValues="true" />
+   <field name="inlinks" type="string" indexed="true" stored="true" multiValued="true" />
```

Comment:
@karanjeets 
While `outlinks` may be a useful addition to the schema, I strongly oppose adding `inlinks` field.
I had brought this discussion multiple times in memex CDR. It looks like no one understood it.

Here is why I am opposing it:
 `inlinks` could potentially grow to a huge list which will overflow the size of multivalued field. 

example:
- Lets checkout the inlinks to `irds.usc.edu`. Lets use an online index for demo.
http://openlinkprofiler.org/r/irds.usc.edu
I see 25 active inlinks to this page (as of now.) We are fine with this.


Now, lets consider a popular page as wikipedia or news site home page?
what would be the count? hundred thousands or a million?

An web page (or image) linked from more than a thousand pages is very easy to hit in large crawl.


Let me know if this makes sense!

cc @sujen1412